examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -160,23 +169,6 @@
               "incoming": 1,
               "outgoing": 1
             }
-          ],
-          "attributes": [
-            {
-              "name": "cluster revision",
-              "code": 65533,
-              "mfgCode": null,
-              "side": "client",
-              "included": 1,
-              "storageOption": "RAM",
-              "singleton": 0,
-              "bounded": 0,
-              "defaultValue": "2",
-              "reportable": 0,
-              "minInterval": 0,
-              "maxInterval": 65344,
-              "reportableChange": 0
-            }
```

Comment:
Not part of this PR but it would be nice for the ZAP code that generate the files to not move around the content of the file when it has not changed. I have not looked at it at all. One possibility for such behavior could be that it uses a JS object {} to store some values and then iterate over it. But those objects are not ordered. Or it could be something else.

Reply:
The ZAP team is aware of this issue and it's in their pipeline to fix this.